package.json

```diff
@@ -7,17 +7,20 @@
     "frontend": "nx serve frontend",
     "build-all": "nx run-many -t build",
     "mini-ci": "nx format:write && nx affected -t test,lint --max-warnings=0",
-    "reload-dm": "nx run-many -t load-dm --skip-nx-cache"
+    "reload-dm": "nx run-many -t load-dm --skip-nx-cache",
+    "start-bot": "npx ts-node --project apps/somnia/src/tsconfig.json apps/somnia/src/main.ts"
   },
   "devDependencies": {
     "@babel/preset-react": "^7.18.6",
     "@emotion/babel-plugin": "11.11.0",
     "@nx/cypress": "16.4.3",
     "@nx/devkit": "16.4.3",
+    "@nx/esbuild": "16.10.0",
     "@nx/eslint-plugin": "16.4.3",
-    "@nx/jest": "16.4.3",
-    "@nx/js": "16.4.3",
+    "@nx/jest": "16.10.0",
```

Comment:
shouldnt have a desync of version of nx package, could cause a lot of problems. would recommend `nx migrate 16.4.3` to sync them up.